searx/static/themes/simple/src/less/style.less

```diff
@@ -33,6 +33,7 @@
   vertical-align: bottom;
   line-height: 1;
   text-decoration: inherit;
+  .ltr-transform();
```

Comment:
> In LTR version:

```less
.ltr-left(@offset) {
  left: @offset;
}
```

> In this RTL version:

```less
.ltr-left(@offset) {
  right: @offset;
}
```

> and so on for the different CSS properties.

Why do we choose the prefix `.ltr-` when it stands for RTL and LTR replacements, isn't it better we use a prefix like `.text-direction-` or _shorter_ `.direction-` or anything other that fits better?  .. The prefix `.rtl-` might confuse here a litle bit.

Reply:
My reasoning:
* We need to describe one version: either the RTL either the LTR version. From this version, the LESS mixins can create the other version.
* `.ltr-left(@offset)` means `left: @offset` in the RTL version --> when `rtl` then `left` (and adjust when `ltr`) --> `.rtl-left`.
  The idea is make clear that `left` applies to the RTL version and it has to be changed in the LTR version.

`direction-left` is telling `left` for the RTL version implicitly or it can tell `left` for LTR version and `right` in the RTL version.